src/ContentInfo/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMemo } from "react";
+import { Navigate, useNavigate, useParams } from "react-router-dom";
+
+import { jsonSchemaValidator } from "@jaam-schema/src";
+
+import { useProjectQuery } from "../@shared/useProjectQuery";
+import { ContentPropertyList } from "./ContentPropertyList";
+import { TimeInfo } from "./TimeInfo";
+import {
+  useContentsState,
+  useSetContentsState,
+} from "../ProjectContents/useContentsStore";
+import { useUpdateContentMutation } from "../ProjectContents/useContentsMutation";
+import * as css from "./index.css";
+
+export function ContentInfo() {
+  const navigate = useNavigate();
+  const params = useParams();
+  const { projectName, schemaName, contentId } = params;
+  const { content: contentState } = useContentsState();
+  const { reset } = useSetContentsState();
+
+  if (!projectName || !contentId || !schemaName) {
+    return <Navigate to="/error" />;
+  }
+
+  const { data: project } = useProjectQuery(projectName);
+
+  if (!project) {
+    return <Navigate to="/error" />;
+  }
+
+  const { schemaList, storageKey: token } = project;
+  const schema = useMemo(
+    () => schemaList.filter(data => data.schemaName === schemaName)[0].schema,
+    [],
+  );
+
+  const updateContent = useUpdateContentMutation({
+    onSuccess: () => {
+      alert("Success content update");
+      reset();
+      navigate(-1);
+    },
+    onError: e => {
+      alert("Failed to update content. Please try again.");
+    },
+  });
+
+  const handlePrevBtnClick = () => {
+    navigate(-1);
+  };
+
+  const handleSaveBtnClick = () => {
+    const { result, message } = jsonSchemaValidator({
+      schema,
+      content: contentState,
+    });
+
+    if (!result) {
+      return alert(message);
+    }
+
+    updateContent.mutate({ token, schemaName, contentId });
+  };
+
```

Comment:
```suggestion
  const updateContent = useUpdateContentMutation();
  
  const handlePrevBtnClick = () => {
    navigate(-1);
  };
  
  const handleSaveBtnClick = () => {
    const { result, message } = jsonSchemaValidator({
      schema,
      content: contentState,
    });

    if (!result) {
      return alert(message);
    }
    
    try {
      await updateContent.mutateAsync({ token, schemaName, contentId });
      alert("Success content update");
      reset();
      navigate(-1);
    } catch {
      alert("Failed to update content. Please try again.");
    }
  };

```

mutateAsync를 쓰면 코드가 더 동기적으로 이해가 잘 될 것 같습니다~